### PR TITLE
Update Readme.txt

### PR DESCRIPTION
**Pull request by vivekvbajpai**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/vivekvbajpai/Academic_Projects/pull/1" target="_blank">ReviewNinja

File: CS546_assignments/HW2/Readme.txt [hunk 1]
```diff
@@ -15,5 +15,5 @@ gcc -fopenmp gauss.c -o gauss
 The openmp code can be executed using folloing command:
 ./gauss 2000
 
-This will run the openmp code using 4 threads for a 2000X2000 matrix.
```
Comment (vivekvbajpai) [inline CS546_assignments/HW2/Readme.txt:18]: wrong

Comment (vivekvbajpai) [inline CS546_assignments/HW2/Readme.txt:18]: incoferefvtc

Comment (vivekvbajpai) [inline CS546_assignments/HW2/Readme.txt:18]: !fix

Comment (vivekvbajpai) [inline CS546_assignments/HW2/Readme.txt:18]: !fixed

**Pull request merged**